src/core/server/http/http_service.ts

```diff
@@ -214,6 +214,10 @@ export class HttpService
     return this.getStartContract();
   }
 
+  private isListening(): boolean {
+    return this.httpServer?.isListening?.();
+  }
```

Comment:
This was this addition necessary (and why the optional chaining given `this.httpServer` is not optional)?